projects/extension/RELEASE_NOTES.md

```diff
@@ -0,0 +1,32 @@
+
+# Pgai Extension Release Notes
+
+# pgai extension 0.4.0 (2024-10-23)
+
+This release add the Vectorizer feature to the extension. Vectorizer is an 
+innovative SQL-level interface for automating the embedding process within
+the database by treating embeddings as a declarative, DDL-like feature—akin to 
+an index. For more details, check out the [documentation](/docs/vectorizer.md).
+
+## Breaking changes
+
+- There are no update paths from 0.1.0, 0.2.0, 0.3.0 to the 0.4.0 release. You 
+  must `DROP EXTENSION ai` and then `CREATE EXTENSION ai VERSION '0.4.0' CASCADE`.
+- The pgai extension is now installed in the `ai` schema. It was previously 
+  installed in the `public` schema by default, but could be explicitly put in 
+  another schema. All pgai functions have moved to the `ai` schema. For example,
+  `openai_list_models()` is now `ai.openai_list_models()`
+- The pg_database_owner and the database user running `CREATE EXTENSION` now get
+  admin privileges over the extension. Other database users/roles will need to
+  be granted privileges to use the extension which is done via functions. 
+  See [privileges.md](docs/privileges.md).
+
+## New Features and Improvements
```

Comment:
```suggestion
### New features and improvements
```

Can you put this section above the breaking changes. Best is to go from good to less good in a RN.